src/main/webapp/instructorHelp.jsp

```diff
@@ -1,228 +1,34 @@
+</c:set>
+<t:helpPage jsIncludes="${jsIncludes}">
   <h1>Help for Instructors</h1>
   <a name="#top"></a>
   <div id="contentHolder">
     <ul>
       <li>
-        <a href="#gs">Getting Started</a>
+        <a href="#students">Students</a>
       </li>
       <li>
-        <a href="#faq">Frequently Asked Questions
-        </a>
-        <div style="margin: 0 auto; padding: 0 50px;">
-          <ul>
-            <li>
-              <a href="#faq-view-student-list-in-course">How do I view a list of students in a course?</a>
-            </li>
-            <li>
-              <a href="#faq-change-details-after-enrolling">How do I change student details after enrolling?
-              </a>
-            </li>
-            <li>
-              <a href="#faq-add-students-to-course">How do I add more students to a course?</a>
-            </li>
-            <li>
-              <a href="#faq-delete-students-from-course">How do I delete students from a course?</a>
-            </li>
-            <li>
-              <a href="#faq-change-googleID-of-student">How do I change the Google ID associated with a student?</a>
-            </li>
-            <li>
-              <a href="#faq-if-courses-displayed-to-students-disappear">What to do if a student says his/her courses have disappeared from the system?</a>
-            </li>
-            <li>
-              <a href="#faq-interpret-contribution-values-in-results">How do I interpret/use contribution values in the results of ‘Team contribution’ type questions?</a>
-            </li>
-            <li>
-              <a href="#faq-if-students-cannot-submit-an-evaluation">What to do if a student says he cannot submit responses due to some technical glitch?</a>
-            </li>
-            <li>
-              <a href="#faq-entered-contribution-differs-from-results">How come the actual contribution values entered by the student is different from the values shown in the results?</a>
-            </li>
-            <li>
-              <a href="#faq-can-student-influence-his-contribution">Can a student influence his ‘perceived contribution’ value by awarding himself a larger contribution?</a>
-            </li>
-            <li>
-              <a href="#faq-how-contribution-numbers-calculated">How are the contribution numbers calculated?</a>
-            </li>
-            <li>
-              <a href="#faq-is-google-account-compulsory-for-student">Is it compulsory for students to use Google accounts?</a>
-            </li>
-            <li>
-              <a href="#faq-if-course-has-no-teams">What if my course doesn’t have teams?</a>
-            </li>
-            <li>
-              <a href="#faq-size-limit-for-course">Is there a size limit for a course?</a>
-            </li>
-          </ul>
-        </div>
+        <a href="#courses">Courses</a>
       </li>
       <li>
-        <a href="#tips">Tips for conducting 'team peer evaluation' sessions</a>
+        <a href="#sessions">Sessions</a>
```

Comment:
Hm, I think I prefer keeping `Responses` information within `Sessions` because the responses to a session seem pretty tightly tied to the session itself. Instead, to consolidate information about responses, I would duplicate the information about viewing student responses under the `Responses` subsection of the `Sessions` section. What do you think?

Reply:
Make sense. I think we can just leave it there.